site/content/pages/organizing/01-Getting started/04-requirements.txt

```diff
@@ -0,0 +1,18 @@
+---
+extension: html
+filter:
+- erb
+- markdown
+title: Rules
+---
+
+Every devopsdays event is different, but there are a few rules to keep in mind if you'd like your event to be listed on devopsdays.org:
```

Comment:
This is an attempt to ensure that people are representing the "devopsdays" brand the way we want them to. Suggestions welcome, but I'm trying to keep the list of "rules" very short.